app/Http/Controllers/App/InboxController.php

```
@@ -51,40 +53,23 @@ public function index($mail = null): Response
     public function import(MailImportRequest $request, $mail): RedirectResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)